examples/plot_2_multidim_sigs.py

```diff
@@ -0,0 +1,249 @@
+"""
+2. 2D and 3D Signals
+====================
+Bycycle supports computing features for 2D and 3D arrays of signals, using
```

Comment:
The first sentence in a doc file should be a short, stand-alone description 

If you go to https://bycycle-tools.github.io/bycycle/v1.0.0rc/auto_examples/index.html, and hover over an example, you can see that these first lines define a pop-up description. Right now, these aren't set up to work well with ByCycle (compare to FOOOF or NeuroDSP). 

It should be:
- title
- blank line
- stand alone description line (to be used as pop-up)
- blank line
- start of main text example

This can be updated across all ByCycle tutorials & examples.

Reply:
Update in 8619a51.